pandas-stubs/core/strings.pyi

```diff
@@ -57,19 +57,14 @@ class StringMethods(NoNewAttributesMixin, Generic[T, _TS]):
     ) -> T: ...
     @overload
     def split(
-        self, pat: str = ..., n: int = ..., *, expand: Literal[True], regex: bool = ...
+        self, pat: str = ..., *, n: int = ..., expand: Literal[True], regex: bool = ...
     ) -> _TS: ...
     @overload
     def split(
-        self, pat: str = ..., n: int = ..., *, expand: bool = ..., regex: bool = ...
+        self, pat: str = ..., *, n: int = ..., expand: bool = ..., regex: bool = ...
     ) -> T: ...
-    @overload
-    def rsplit(
-        self, pat: str = ..., n: int = ..., *, expand: Literal[True], regex: bool = ...
-    ) -> T: ...
```

Comment:
why remove the above overload?

Reply:
The resplit overload appears to be erdundant.  Both return `T`, unlike split which returns `_TS`. Maybe a bug in the `rsplit` overload?

Reply:
Yes, I think that is the case.  Can you fix that?

Reply:
Fixed.